hudi-utilities/src/test/java/org/apache/hudi/utilities/streamer/TestStreamSync.java

```diff
@@ -91,7 +93,10 @@ void testFetchNextBatchFromSource(Boolean useRowWriter, Boolean hasTransformer,
                                     Boolean isNullTargetSchema, Boolean hasErrorTable, Boolean shouldTryWriteToErrorTable) {
     //basic deltastreamer inputs
     HoodieSparkEngineContext hoodieSparkEngineContext = mock(HoodieSparkEngineContext.class);
-    HoodieStorage storage = new HoodieHadoopStorage(mock(FileSystem.class));
+    HoodieStorage storage = HoodieStorageUtils.getStorage(
```

Comment:
It would be nice if we can also use `HoodieStorageUtils` to get storage instance in tests like this(not with the `FileSystem`). But I'm not sure how feasible it is. If it's quite tricky then we can just use `HoodieHadoopStorage` constructor for tests